conf/application.conf

```diff
@@ -136,36 +135,30 @@ aws {
   sqs {
     queue.inbound =  "SET THE QUEUE"
     queue.inbound = ${?AWS_SQS_QUEUE_INBOUND}
-    retry.interval = 20 seconds
+    retry.interval = "20 seconds"
   }
 }
+
 clam.antivirus {
-# To run locally, override with -Dclam.antivirus.host=avscan
-  host = localhost
+  host = avscan
   port = 3310
   timeout = 60000
 }
+
 logger {
+  application = "INFO"
```

Comment:
We should set the default logger level to WARN, as this is what it should be in Production. Other environments can override as required.